scripts/email/templates/versioned_preenroll/sp25/current_freshman.py

```diff
@@ -0,0 +1,136 @@
+import sys
```

Comment:
do you mind clarifying the difference between this python file and the changes in scripts/email/playground/template.html ?

Reply:
Yes certainly.

- Hosting images differently
- Some inline CSS changes, e.g. no use of negative margins as that gets stripped by Gmail
- Actual load in the template variables instead of leaving them in square brackets
- Footer updates I made due to again CSS limitations on email clients

I can resync the template to be similar to the `current_freshman.py` file, but it's really just a playground and would be annoying having to update everything in two places going forward.